src/main/kotlin/minesweeper/domain/Game.kt

```diff
@@ -0,0 +1,58 @@
+package minesweeper.domain
+
+import minesweeper.domain.GameState.PROGRESS
+import minesweeper.domain.GameState.SKIP
+import minesweeper.domain.GameState.TERMINATE
+
+class Game(private val board: MineBoard) {
+
+    private val rows = board.rowSize()
+    private val columns = board.columnSize()
+
+    fun open(targetPositionText: String): GameState {
+        val index: Int = Position.toIndex(columns, targetPositionText)
+        val target: Coordinate = board.coordinates[index]
+
+        if (target.isMine()) return TERMINATE
+        if (target.count > 0) {
+            target.open()
+            return SKIP
+        }
+
+        openAround(index)
+
+        return PROGRESS
+    }
+
+    private fun openAround(index: Int) {
+        val positionArray = Position.values()
```

Comment:
```suggestion
        val positions = Position.values()
```

array라는 이름이 아닌 복수형이 더 괜찮지 않을까요?